test/fixtures/onChain.js

```diff
@@ -105,10 +105,10 @@ export default [
 
   // still poor test
   // TODO: init state with returnData first
-  { code: ['00', OP.RETURNDATASIZE], step: 0 },
+  { code: [OP.STOP, OP.RETURNDATASIZE], step: 1 },
```

Comment:
seems to be related to first comment. is it that stop opcode should increase step counter?

Reply:
Right :)